File: Java/Important Fundamentals/16. Fibonacci Series upto nth term.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.println("Enter the Fibonacci value range:");
        int fibonacciValue = userInput.nextInt();

        int n1 = 0;
        int n2 = 1;
        System.out.println(n1+ " " +n2);

        for( int i=2; i<fibonacciValue; i++ ) {
            int n3 = n1 + n2;
            n1 = n2;
            n2 = n3;
            System.out.println(n3+" ");
        }
        userInput.close();
    }
}

File: Python/01. Important Fundamentals/16. Fibonacci Series upto nth term.py
number = int(input("Enter a number to print the FIbonacci:"))
n1 = 0
n2 = 1

print(n1,n2, end=" ")
for i in range(2,number):
    n3 = n1 + n2
    n1 = n2
    n2 = n3
    print(n3, end=" ")